Sources/Castor/Castor.docc/Articles/user-interfaces/standard-views-article.md

```diff
@@ -0,0 +1,102 @@
+# Standard Views
+
+@Metadata {
+    @PageColor(purple)
+    @PageImage(purpose: card, source: standard-views-card, alt: "An image depicting Google Cast buttons and player views.")
+}
+
+User friendly default views.
+
+## Overview
+
+``Castor`` provides modern SwiftUI views that are easier to integrate. These views also come with APIs to create fully custom player interfaces if you prefer to design your own experience.
+
+## Castor standard views
+
+Castor includes several built-in views to cover common Cast use cases. Below are examples for the main components: ``CastButton``, ``CastMiniPlayerView``, and ``CastPlayerView``.
+
+> Tip: By default, the provided views uses the `.accentColor`. You can change this color using the `.foregroundStyle` modifier, but note that this will not affect the color within the child views or the ``SwiftUI/Slider`` color. To update the default `accentColor` in the view displayed by the ``CastButton``, use the `.tint` modifier.
```

Comment:
```suggestion
> Tip: By default, the provided views use the `.accentColor`. You can change this color using the `.foregroundStyle` modifier, but note that this will not affect the color within the child views or a ``SwiftUI/Slider`` color. To update the default `accentColor` in the view displayed by the ``CastButton``, use the `.tint` modifier.
```